engine/main.cc

```diff
@@ -151,6 +150,28 @@ int main(int argc, char* argv[]) {
       file_manager_utils::cortex_data_folder_path = argv[i + 1];
     } else if (strcmp(argv[i], "--port") == 0) {
       server_port = std::stoi(argv[i + 1]);
+    } else if (strcmp(argv[i], "--loglevel") == 0) {
+      std::string log_level = argv[i + 1];
+      if (log_level == "TRACE") {
```

Comment:
It would be better if we have a helper function for this, like `helper::ToLogLevel(const std::string&)`